etc/env/file_based/genesis.yaml

```diff
@@ -1,19 +1,20 @@
-bootloader_hash: 0x010007ede999d096c84553fb514d3d6ca76fbf39789dda76bfeda9f3ae06236e
-default_aa_hash: 0x0100055b041eb28aff6e3a6e0f37c31fd053fc9ef142683b05e5f0aee6934066
-genesis_root: 0x1f50e4eda06a68d96a0272ba4581a342df2227ad12c23759ab7d78157950e69a
-genesis_batch_commitment: 0x84d7b576b9374729e0b38439c97aaf5074335e2a8d1c7a2e4581c1c1ec611631
-genesis_rollup_leaf_index: 46
-genesis_protocol_version: 22
-l1_chain_id: 9
-l2_chain_id: 270
-fee_account: 0x0000000000000000000000000000000000000001
-l1_batch_commit_data_generator_mode: Rollup
+genesis_root: 0xb749a312a6f3ece4d43c5a02730587e133fd4ff4f90edf27d866f49946302abe
+genesis_rollup_leaf_index: '46'
+genesis_batch_commitment: 0x7c15b7e70795ba6d07ffa67482343d6b07fad0941d4fd7fe8a748b4da764c006
+genesis_protocol_version: 24
+default_aa_hash: 0x0100055be0b42ec7b676a62a67507bad5db7b30ccbce9640ba9db9e37cc8d499
+bootloader_hash: 0x010008095dfd6a873af3909665f4d7929d3bf86baf725277618f85135e4054d2
+l1_chain_id: '9'
+l2_chain_id: '270'
+fee_account: '0x0000000000000000000000000000000000000001'
 prover:
-  recursion_leaf_level_vk_hash: 0x400a4b532c6f072c00d1806ef299300d4c104f4ac55bd8698ade78894fcadc0a
-  recursion_node_level_vk_hash: 0x5a3ef282b21e12fe1f4438e5bb158fc5060b160559c5158c6389d62d9fe3d080
   recursion_scheduler_level_vk_hash: 0x063c6fb5c70404c2867f413a8e35563ad3d040b1ad8c11786231bfdba7b472c7
+  recursion_node_level_vk_hash: 0x5a3ef282b21e12fe1f4438e5bb158fc5060b160559c5158c6389d62d9fe3d080
+  recursion_leaf_level_vk_hash: 0x400a4b532c6f072c00d1806ef299300d4c104f4ac55bd8698ade78894fcadc0a
+  recursion_circuits_set_vks_hash: '0x0000000000000000000000000000000000000000000000000000000000000000'
```

Comment:
Why some fields are in '' and some are not?

Reply:
so... it's a good question. 
It is how automatical tools work, if it could fit into u64 it must be in quotes, but if it's not it could be non-in quotes.